problemsets-mat223/linearalbegra.tex

```diff
@@ -5207,8 +5225,23 @@ \section*{Eigenvectors}
 	
 
 	\begin{parts}
-		\item Give an eigenvector for $T$.  What is the eigenvalue?
+		\item Give an eigenvector for $T$. What is the eigenvalue?
+			\begin{solution}
+				$\vec v=\mat{1\\1}$ is an eigenvector for $T$, with corresponding
+				eigenvalue $\frac{3}{2}$. 
+
+				We can see from the image that 
+				$T\mat{1\\1}=\mat{\frac{3}{2}\\\frac{3}{2}}=\frac{3}{2}\mat{1\\1}$.
+			\end{solution}
 		\item Can you give another?
+			\begin{solution}
+				Any scalar multiple of $\mat{1\\1}$ is also an eigenvector for $T$.
```

Comment:
Include "but more interestingly, `\mat{-1\\1}` is an eigenvector with eigenvalue ...`

Reply:
Done. I'm going to include an explanation for why it's an eigenvector, but let me know if you actually want that.